audio_context.html

```diff
@@ -0,0 +1,151 @@
+<!doctype html>
+<head>
+  <title>Web Audio Api - Audio Context</title>
+  
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+   
+  <style>
+  
+    :root {
+      --flex-direction: column;
+      --line-height: 4em;
+    }
+    
+    @media(min-width: 250px) {
+      :root {
+        --flex-direction: row;
+        --line-height: 8em;
+      }
+    }
+       
+    h1{
+      text-align: center;
+    }
+      
+    main {
+      outline: thick double;
+      background-color: whitesmoke;
+    }
+       
+    fieldset {
+      display: flex;
+      border: none;
+      background: yellow;
+    }
+    form > fieldset {
+      flex-direction: var(--flex-direction);
+    }
+    fieldset fieldset {
+      background: red;
+    }
+    fieldset input[type="range"] {
+      width: 20vw;
+      display: inline-block;
+      top: 0.3em;
+      position: relative;
+    }
+    
+    form > fieldset > label {
+      background: turquoise;
+      line-height: var(--line-height);
+    }
+    form > header {
+      background:yellow;
+    }
+    
+    [type="checkbox"] {
+    }
+  
+
+  </style>
+ 
+  <script></script>
+</head>
+
+ 
+<body>
+  <header>
+    <h1>DevPunks Sequencer</h1>    
+  </header>
+
+<main>
+  <form action=#>
+    <header>DevPunks Sequencer
+      <label>BPM
+        <input type=range min=10 max=522 value=251>
+      </label>
+  
+      <label>Play
+        <input name="Play" type="checkbox">
+      </label>
+    </header>
+ 
+<fieldset><legend>Sweep</legend>
+  <label>Attack
+    <input type="range"name="Attack" min="0" max="1" value="0.2" step="0.1">
+  </label>
+
+  <label>Release
+    <input type="range" name="Release" min="0" max="1" value="0.5" step="0.1">
+  </label>
+
+  <fieldset><legend hidden>Checkboxes</legend>
+    <label><input name="Pad1" type="checkbox"></label>
```

Comment:
`name`s for checkboxes +radio  buttons should be the same. This is how html links value(s) to key in form submission.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#name